electron-test/wdio.conf.ts

```diff
@@ -116,7 +119,12 @@ export const config: Options.Testrunner = {
     // Services take over a specific job you don't want to take care of. They enhance
     // your test setup with almost no effort. Unlike plugins, they don't add new
     // commands. Instead, they hook themselves up into the test process.
-    services: ['electron'],
+    services: [
+        'electron',
+        [ServerLauncher, {
+            someOption: true
```

Comment:
I don't think we need the options, do we?
I guess you got it from that [example](https://webdriver.io/docs/customservices/#import-service-from-module)?

Reply:
Yes, I was following the instructions of the example. We can remove the option to reduce code or let it.

@axi92 Which option do you think is best? I'm according with both.

Reply:
I agree this can be removed as it seems to be just a placeholder option put there as an example

Reply:
Done.